general/CMakeLists.txt

```diff
@@ -27,12 +31,18 @@ list(APPEND SRCS
 
 list(APPEND HDRS
   array.hpp
+  binaryio.hpp
```

Comment:
I think this might be a leftover from the git merge.

Reply:
This is old. I updated it a while ago in f46b034997ef.